components/camel-flowable/pom.xml

```diff
@@ -0,0 +1,110 @@
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>components</artifactId>
+        <version>4.9.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>camel-flowable</artifactId>
+    <packaging>jar</packaging>
+    <name>Camel :: Flowable</name>
+    <description>Camel Flowable support</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-core</artifactId>
+        </dependency>
+        <dependency>
+      		<groupId>org.apache.camel</groupId>
+      		<artifactId>camel-spring</artifactId>
+    	</dependency>
+
+        <!-- testing -->
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-test-spring-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+        
+        <!-- Flowable JARs -->
+        <dependency>
+            <groupId>org.flowable</groupId>
+            <artifactId>flowable-engine</artifactId>
+            <version>${flowable-version}</version>
+        </dependency>
+        <dependency>
+      		<groupId>org.flowable</groupId>
+      		<artifactId>flowable-spring</artifactId>
+      		<version>${flowable-version}</version>
+    	</dependency>
+    	<dependency>
+      		<groupId>org.flowable</groupId>
+      		<artifactId>flowable-event-registry-spring</artifactId>
+      		<version>${flowable-version}</version>
+    	</dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <version>${log4j2-version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+      		<groupId>org.springframework</groupId>
+      		<artifactId>spring-test</artifactId>
+      		<scope>provided</scope>
+      		<version>${spring-version}</version>
+    	</dependency>
+    	<dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-jdbc</artifactId>
+            <scope>provided</scope>
+            <version>${spring-version}</version>
+        </dependency>
+    	<dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <scope>test</scope>
+            <version>${hikari-version}</version>
+        </dependency>
+    	<dependency>
+      		<groupId>com.h2database</groupId>
+      		<artifactId>h2</artifactId>
+      		<scope>test</scope>
+      		<version>${h2-version}</version>
+    	</dependency>
+    	<dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+            <version>${junit-version}</version>
+        </dependency>
```

Comment:
do we need JUnit 4?  We are using 5.

Reply:
Testing should be junit5 based, and should use camel-test-junit5 for testing so its testing the standard Camel way and its possible for anyone to help maintain this code in the future

Reply:
this was changed also together with removing the dependency to Spring